package.json

```diff
@@ -1,6 +1,16 @@
 {
+  "scripts": {
+    "build": "lerna run build:js && ttsc --build --verbose",
+    "build:clean": "rimraf ./packages/**/dist",
+    "build:watch": "ttsc --build && lerna run build:watch --parallel",
+    "test": "lerna run test --stream -- --colors --maxWorkers=4",
+    "lint": "eslint packages",
+    "lint:fix": "eslint --fix packages"
+  },
```

Comment:
I actually leave these at the bottom because I run `cat package.json` to see what I can run.

Reply:
And I guess you're doing this for the exact reason, but you're opening it in an editor and reading from the top.